test/test_shared_library.py

```diff
@@ -0,0 +1,98 @@
+from translator.source_code import translator_shared_library
```

Comment:
Missing a test for the `initialize_info` method.

Reply:
added unit test for initialize_info